catroid/gradle/intellij_config_tasks.gradle

```diff
@@ -29,6 +29,7 @@ task gitSkipWorktreeForIntellijConfigFiles << {
         'git update-index --skip-worktree Catroid.iml'.execute().text.trim()
         'git update-index --skip-worktree catroid/src/test/catroidSourceTest.iml'.execute().text.trim()
         'git update-index --skip-worktree .idea/codeStyleSettings.xml'.execute().text.trim()
+        'git update-index --skip-worktree catroid/catroid.iml'.execute().text.trim()
```

Comment:
Hey Oh! :wave: :smile:

Do you know gradle well enough to do a small improvement on this task? [CAT-2258](https://jira.catrob.at/browse/CAT-2258)
I would love to see `gitSkipWorktreeForIntellijConfigFiles` and `gitNoSkipWorktreeForIntellijConfigFiles` using and sharing a list.

If not, no problem. Just update the `gitNoSkipWorktreeForIntellijConfigFiles` as well. :smiley_cat: